b2share/modules/records/serializers/schemas/datacite.py

```diff
@@ -119,6 +119,10 @@ def get_contributors(self, obj):
                 for c in obj['metadata'].get('contributors', [])]
 
     def get_descriptions(self, obj):
+        datacite_v3_description_types = set([
+            "Abstract", "Methods", "SeriesInformation", "TableOfContents"])
```

Comment:
I would put the set creation out of the function. No need to recreate it each time. You can put an underscore first in order to avoid polluting the module's namespace `_datacite_v3_description_types `.